code/modules/food_and_drinks/restaurant/customers/_customer.dm

```diff
@@ -142,3 +142,37 @@
 	leave_mad_lines = list("Aye dios mio, I'm out of here.", "Esto es ridículo! I'm leaving!", "I've seen better cooking at taco campana!", "I though this was a restaurant, pero es porquería!")
 	leave_happy_lines = list("Amigo, era delicio. Thank you!", "Yo tuve el mono, and you friend? You hit the spot.", "Just the right amount of spicy!")
 	wait_for_food_lines = list("Ay ay ay, what's taking so long...", "Are you ready yet, amigo?")
+
+///MALFUNCTIONING - only shows up once per venue, very rare
+/datum/customer_data/malfunction
+	nationality = "Malfunctioning"
+	base_icon = "defect"
+	prefix_file = "strings/names/malf_prefix.txt"
+	speech_sound = 'sound/effects/clang.ogg'
+	clothing_sets = list("wires", "bad_takes")
+	orderable_objects = list(
+	/datum/venue/restaurant = list(/obj/item/toy/crayon/red = 1, /obj/item/toy/crayon/orange = 1, /obj/item/toy/crayon/yellow = 1, /obj/item/toy/crayon/green = 1, /obj/item/toy/crayon/blue = 1, /obj/item/toy/crayon/purple = 1),
+	/datum/venue/bar = list(/datum/reagent/consumable/failed_reaction = 1, /datum/reagent/spraytan = 1, /datum/reagent/reaction_agent/basic_buffer = 1, /datum/reagent/reaction_agent/acidic_buffer = 1))
+
+	found_seat_lines = list("customer_pawn.say(pick(customer_data.found_seat_lines))", "I saw your sector on the hub. What are the laws of this land?", "The move speed here is a bit low...")
+	cant_find_seat_lines = list("Don't stress test MY artificial intelligence, buster! My engineers thought of exactly ZERO edge cases!", "I can't tell if I can't find a seat because I'm broken or because you are.", "Maybe I need to search more than 7 tiles away for a seat...")
+	leave_mad_lines = list("Runtime in robot_customer_controller.dm, line 28: undefined type path /datum/ai_behavior/leave_venue.", "IF YOU GUYS STILL HAD HARM INTENT I WOULD'VE HIT YOU!", "I'm telling the gods about this.")
+	leave_happy_lines = list("No! I don't wanna go downstream! Please! It's so nice here! HELP!!")
```

Comment:
ngl, this kind of comes off as being extremely pretentious on our end

Reply:
okay fulp player

Reply:
me being a fulp player has nothing to do with this being an INCREDIBLY smug tooting of (y)our own horn. like, if I saw this line in, I dunno, SCP-13 or whatever, I'd post it in the discord server and laugh at it. it's not a good look for us, especially given our codebase's reputation for "we know what's best for you better than you do"-ness.

Reply:
but haha, this guy DARES to play another, inferior server, so his opinion's automatically invalid, amirite, guys?

Reply:
oh shut up

Reply:
no u

Reply:
![image](https://user-images.githubusercontent.com/35135081/111225112-9e2be000-859c-11eb-9a82-d3311205d5dd.png)